src/components/Dialogue/_Dialogue.scss

```diff
@@ -0,0 +1,81 @@
+.dlg-box {
+  height: 40vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  img {
+    width: 10%;
+  }
+}
+
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100vw;
+  height: 100vh;
+  color: #fff;
+}
+
+.modal-overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  cursor: pointer;
+}
+
+.modal-box {
+  height: fit-content;
+  position: relative;
+  background-color: #383838;
+}
+
+.modal-title {
+  font-weight: 500;
+  font-size: 1.2rem;
+  padding: 1rem;
+  cursor: default;
+}
+
+.modal-list {
+  list-style-type: none;
+  padding: 0;
+  width: 100%;
+}
+
+.list-content {
+  font-weight: 300;
+  cursor: pointer;
+  width: 100%;
+  padding: .34rem .7rem;
+
+  &:hover {
+    background-color: #585858;
+  }
+}
+
+.dlg-icon {
+  box-sizing: border-box;
+  margin-right: .8rem;
+  width: 2.5rem;
+  height: 2.5rem;
+  padding: .3rem;
+  border-radius: 50%;
+  background-color: #bbdefb;
+}
+
+.dlg-add {
+  background-color: #757575;
+}
+
+.alert-title {
+  border: solid red 2px;
+}
```

Comment:
seems like Patrick wanted us to use variables more for SASS, I am still studying it too.

Reply:
This was just to see if I had the right element, I deleted it now.